### DISCUSSION TITLE
Different site icons

### DISCUSSION THREAD

**Discussion by andrewfairlie** [craftcms/cms#8559] [Ideas]
### Description
We've noticed that translators often get confused and populate the English content with German content (for example).

Anything the CMS can do to make the site/locale you're editing clearer would be great and I think a good way to do this would be for different sites to have different sidebar icons. That way we could upload a big flag for the country the site represents to make it really clear to the user.

**Comment by KatieMFritz** [comment]
@andrewfairlie This might be too much work for what you're looking for, but I add custom styles (header bar color coding) for multisite with this process:
https://gist.github.com/KatieMFritz/02808d01203e2d566d98689b5315b8d8
